pcsx2/x86/ix86-32/iR5900-32.cpp

```diff
@@ -1113,76 +1113,23 @@ void iFlushCall(int flushtype)
 //	}
 //}
 
-
-static u32 scaleBlockCycles_helper()
+static u32 scaleblockcycles()
 {
-	// Note: s_nBlockCycles is 3 bit fixed point.  Divide by 8 when done!
-
-	// Let's not scale blocks under 5-ish cycles.  This fixes countless "problems"
```

Comment:
I think I would not remove that comment. Since you follow it with bool lowcycles later.